infra/dockerfile.commit_retrieval

```diff
@@ -16,6 +16,6 @@ RUN apt-get update && \
     apt-get autoremove -y && \
     rm -r /var/lib/apt/lists/*
 
-COPY infra/hgrc /root/.hgrc
+COPY infra/hgrc /etc/mercurial/hgrc.d/bugbug.rc
```

Comment:
Could you apply the same change to .taskcluster.yml (the `tests_task` task), so we make sure we run the tests in the same configuration

Reply:
Done